_docs/v2.1/setup/install.md

```diff
@@ -67,6 +67,12 @@ steps below:
      python install/gcp_installer.py
      ```
 
+     If you don't plan to share your Forseti instance with other people, 
```

Comment:
We should try to clarify "people".  Either something like `non-admin` or "If forseti users already have the same permission as the server".

Reply:
Updated.